### PR TITLE
Add docs about storage

### PR DESCRIPTION
**Pull request by Pana**
1. temporary remove intro about Conflux studio
2. Remove old RPC urls

### CODE REVIEW

File: docs/introduction/en/storage.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Entry | Key/Address({0,1}<sup>256</sup>)   | value({0,1}<sup>256</sup>)  |
+|  :----:  |  :----:  | :----:  |
+1| 0...00000  | 0 |
+2| 0...00001  | 1 |
+3| 0...00002 | 2 |
+...| ... | ... |
+2<sup>256</sup>| f...fffff| 0 |
+
+Since 256bits=32bytes, the corresponding length of two 256bits is: 32+32=64 (Bytes)
```
Comment (Thegaram) [inline docs/introduction/en/storage.md:31]: Again, I think there's some confusion here. The "key" part is actually not stored in storage, only the value part. Each storage slot will store the data (32B, same as Ethereum) plus the owner (20B), together they are stored as 64B. Let me discuss this with Chenxing a bit before we update this part.

File: docs/introduction/en/storage.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Summary of common variables and their corresponding entries in Solidity:
+
+|   Variable   |                            Length                             |                      Definition Method                       |
+| :------: | :---------------------------------------------------------: | :-------------------------------------------------: |
+| Ordinary variables |                   1 ordinary variable corresponds to 1 entry                    |                uint public count=0;                 |
+| mapping  |              Each key of mapping corresponds to 1 entry              |      mapping(address => uint) public balances;      |
+|  array   | Each element of the array corresponds to 1 item, and the length of the array arr.length is an additional item | uint[5] fixedArr = [1,2,3,4,5]; string productname; |
```
Comment (Thegaram) [inline docs/introduction/en/storage.md:39]: >  Each element of the array corresponds to 1 item, and the length of the array arr.length is an additional item

I think array elements can be "packed tightly" (e.g. with `uint128`). But we do not need to go into so many details in this doc.

File: docs/introduction/en/storage.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+|  struct  |              Accumulation of the number of entries corresponding to each field in the struct              |  struct Person {uint age;uint stuID;string name;}   |
+
+
+## How to use storage mortgage
```
Comment (Thegaram) [inline docs/introduction/en/storage.md:43]: > storage mortgage

storage collateral

File: docs/introduction/en/storage.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+When users send a Conflux transaction, they need to fill in a `storageLimit` field (in bytes). The upper limit stipulates that the deposit increase of the deposit payer before and after the transaction is executed shall not exceed the `storage upper limit` multiplied by 1/1024 CFX. If this value is filled in too low, it will cause the deposit to exceed the upper limit after execution and the execution will fail. Therefore, in general, this field needs to be set to a value larger than the actual usage, and the excess part will not generate storage mortgage. However, it is not recommended to fill in too high, because it may cause the sender's balance to be insufficient to pay the deposit, which may result in transaction failure. Fullnode provides the RPC method `cfx_estimateGasAndCollateral` to estimate the storage size that a transaction will use.
+
```
Comment (Thegaram) [inline docs/introduction/en/storage.md:46]: Maybe add: "Storage limit plays the same role for storage as gas limit does for execution."

File: docs/introduction/en/storage.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+After the transaction is executed, the `Receipt` contains several fields about storage changes:
+
+* `storageCollateralized` This is the amount of data that is stored and mortgaged
+* `storageCoveredBySponsor` Whether the storage mortgage for this transaction is sponsored by the sponsor
+* `storageReleased` The storage released by this transaction
+
+You can use the `cfx_getCollateralForStorage` method to query the storage size currently mortgaged by an address, the unit is bytes, and the value divided by 1024 is the amount of CFX storage mortgaged. In addition, it can also be obtained through the `cfx_getAccount` method, and the returned information contains the `collateralForStorage` field.
```
Comment (Thegaram) [inline docs/introduction/en/storage.md:53]: > mortgaged

collateralized

**Review by Thegaram — CHANGES_REQUESTED** [review]

File: docs/introduction/en/storage.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+In the Conflux network, each storage entry occupies a space of 64B (B is Bytes, byte), which is also the size of the key/value pair in the world state. It should be noted that the key in the blockchain is generally 256bits long and the value It is also 256bits long (each is 32B long, and the total is 64B long). The deposit required for storage is proportional to the minimum multiple of 64B that can cover all stored items. For each storage entry, the last account that writes to the entry is called the owner of the storage entry. If a storage item is written during the execution of contract C, and a guarantor provides guarantee for it, then C is regarded as the writer of the item and accordingly becomes the owner of the item (see section 7.1 for details). In the world state, during the entire life cycle of a storage item, the owner of the item must lock a fixed amount of CFX as a storage deposit for the storage space. Specifically, for each storage entry with a size of 64B, its owner will be locked by 1/16CFX. For occupying `1KB` space, you will pay `1CFX` as a deposit, and the corresponding formula is as follows:
```
[the remark this answers is not in the document]
Comment (Thegaram) [reply docs/introduction/en/storage.md:11]: Seems like I misunderstood this. I don't think this description is very clear but we can keep it as it is for now.

File: docs/introduction/en/storage.md (hunk 1 above)
Comment (Thegaram) [reply docs/introduction/en/storage.md:31]: This part is correct, we can keep it as it is.

File: docs/introduction/en/storage.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+![](/img/storage-formula.png)
```
[the remark this answers is not in the document]
Comment (Pana) [reply docs/introduction/en/storage.md:13]: I have test it, it's ok in dark mode

File: docs/introduction/en/storage.md (hunk 2 above)
Comment (Pana) [reply docs/introduction/en/storage.md]: Yeah, i have remove this part

File: docs/introduction/en/storage.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+---
+id: conflux_storage
+title: Storage
+custom_edit_url: https://github.com/Conflux-Chain/conflux-developer-site/edit/master/docs/introduction/en/storage.md
+keywords:
+  - storage
+---
+
+Conflux introduced the Collateral for storage (CFS) mechanism as a pricing method for using storage. Compared with the one-time storage fee in Ethereum, the CFS mechanism will be more fair and reasonable. In principle, this mechanism requires a certain amount of funds to be locked as collateral to occupy storage space. Before the corresponding storage space is released or covered by others, the collateral will be locked, and the corresponding interest generated by the locked collateral will be directly allocated to the miners for the maintenance of the storage space. Therefore, the storage cost of Conflux also depends on the length of time the storage space is occupied. Conflux specifically introduced this mechanism in detail in chaper 7of its [Conflux_Protocol_Specification](https://confluxnetwork.org/files/Conflux_Protocol_Specification_20201020.pdf).
```
Comment (Thegaram) [inline docs/introduction/en/storage.md:9]: > chaper 7of 

chaper 7 of

**Review by Thegaram — APPROVED** [review]

**Pull request merged**